app/uk/gov/hmrc/transitmovements/controllers/MovementsController.scala

```diff
@@ -125,7 +122,7 @@ class MovementsController @Inject() (
           arrivalData.generationDate,
           received,
           None,
-          ObjectStoreResourceLocation(objectSummary.location.asUri),
+          ObjectStoreURI(objectSummary.location.asUri),
```

Comment:
This seems to flip-flop between the 2?  I think we have ObjectStoreResourceLocation in common-transit-convention-traders-

Reply:
It should be both:

* `ObjectStoreURI` contains the entire URI to object-store
* `ObjectStoreResourceLocation` is basically the same, but with the owner.

When this was introduced, it's been used for both the URI and the owner-less URI, and it was causing issues with storing and retrieving. The whole idea of using the `AnyVal` wrappers is to ensure the right string goes to the right place, but unfortunately, due to the fact this was used in two different ways, I split them like this.

Because the object-store code has to be rationalised into `transit-movements` anyway, that problem should go away elsewhere.